daemon/daemon_main.go

```diff
@@ -581,6 +581,9 @@ func init() {
 	flags.Bool(option.K8sRequireIPv6PodCIDRName, false, "Require IPv6 PodCIDR to be specified in node resource")
 	option.BindEnv(option.K8sRequireIPv6PodCIDRName)
 
+	flags.Uint(option.K8sServiceCacheSize, defaults.K8sServiceCacheSize, "Cilium service cache size for kubernetes")
```

Comment:
@fristonio can you please use `flags.MarkHidden` for this flag?

Reply:
I have updated the pull request.